### DISCUSSION TITLE
Can I allow Page.go_to take additional wait parameter?

### DISCUSSION THREAD

**Discussion by erickgskovik** [rubycdp/ferrum#394] [General]
Hey,

I am working on rendering local HTML with Ferrum. During the implementation, I realize `Page.go_to` doesn't wait. And it might not be enough time for rendering to complete. I found `FERRUM_GOTO_WAIT` as a work around. It can be better to also accept `wait`.

`Page.goto` accepts an optional [url](https://github.com/rubycdp/ferrum/blob/502655a86b4693096c6ec07daa711f9544c55440/lib/ferrum/page.rb#L113). Can it also accept a wait parameter?

For the implementation details,  `Page.go_to` accepts `wait`. `FERRUM_GOTO_WAIT` will be set if `wait` parameter is not set.

Another question to the details is that is there a policy to accept slowmoable? For example, for the Page.go_to.

**Comment by route** [comment]
1) `Page.go_to` returns when the page is ready do work with, it's fully loaded, the DOM is rendered. In the browser there's no clear moment the page is "fully loaded" since everything is async.
2) If we put `slowmoable` to every command then we would see "frozen" browser because there are a lot of commands that don't have visible representation, so we put slowmoable only to commands that have something visible, like: click, type, scroll, etc
